server/events/vcs/gitlab_client.go

```diff
@@ -295,7 +295,7 @@ func (g *GitlabClient) WaitForSuccessPipeline(ctx context.Context, pull models.P
 
 // MergePull merges the merge request.
 func (g *GitlabClient) MergePull(pull models.PullRequest, pullOptions models.PullRequestOptions) error {
-	commitMsg := common.AutomergeCommitMsg
+	commitMsg := common.AutomergeCommitMsg(pull.Num)
```

Comment:
What about for other vcs like github, bitbucket, etc?

Reply:
Both github and bitbucket are using their default commit messages. This change would not affect them.